Source/WebKit/Shared/RemoteLayerTree/RemoteScrollingCoordinatorTransaction.cpp

```diff
@@ -557,6 +568,46 @@ bool ArgumentCoder<SnapOffset<float>>::decode(Decoder& decoder, SnapOffset<float
 }
 
 
+void ArgumentCoder<KeyboardScroll>::encode(Encoder& encoder, const KeyboardScroll& scroll)
```

Comment:
We should be able to use `WebCoreArgumentCoders.serialization.in` rather than manually writing the encoder/decoder.

Reply:
@pxlcoder ah ok, didn't know that was a thing! I update my PR, lmk if I did it correctly!

Reply:
It's a relatively new addition 😃